viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -144,6 +145,19 @@ export class ComboControls extends EventDispatcher {
     };
   }
 
+  temporaryDisable(value: boolean): void {
+    this._enabled = value;
+  }
+
+  set enabled(value: boolean) {
+    this._enabled = value;
+    this.enabledCopy = value;
+  }
+
+  get enabled(): boolean {
+    return this._enabled;
+  }
```

Comment:
See comment below, this is required because I want to store "ground truth" enabled and return to it after a few disables for controls. It can be done cleaner by just having another variable called "temporary disable" that is totally independent of controls.enable.